### PR TITLE
ARROW-3276: [Packaging] Add support for Parquet deb/rpm packages

### PR DESCRIPTION
**Pull request by kou**

### CODE REVIEW

**Comment by kszucs** [comment]
Great! Thanks @kou!

How should we handle parquet versioning? Will we have distinct git tags for parquet as well as separate release?

**Comment by kou** [comment]
Do you mean that Apache Arrow and Apache Parquet C++ use different versions?
I thought that "Apache Arrow's release archive ships Apache Parquet C++" means "Apache Arrow and Apache Parquet C++ use the same version"...

If we use the different versions for Apache Arrow and Apache Parquet C++, we should create two `debian/` directories and two `*.spec` files for Apache Arrow and Apache Parquet C++.

**Comment by wesm** [comment]
> How should we handle parquet versioning? Will we have distinct git tags for parquet as well as separate release?

This is up to the Parquet community. To avoid complications we may just want to bundle the Parquet libraries in the same package as libarrow.so

**Comment by kszucs** [comment]
Just asking because the two most recent parquet-cpp version numbers are `1.4.1`, `1.5.0pre` and incrementing it to `0.11.0` can cause confusion.

**Comment by wesm** [comment]
My thought was to bundle everything as a unit, to avoid having to worry about release coordination between Apache Arrow and Apache Parquet

File: dev/tasks/linux-packages/debian.ubuntu-trusty/control [hunk 1]
```diff
@@ -42,6 +44,30 @@ Description: Apache Arrow is a data processing library for analysis
...
 
+Package: libparquet11
+Section: libs
+Architecture: any
+Multi-Arch: same
+Pre-Depends: ${misc:Pre-Depends}
+Depends:
+  ${misc:Depends},
+  ${shlibs:Depends}
+Description: Apache Parquet is a columner storage format
```
Comment (kszucs) [inline dev/tasks/linux-packages/debian.ubuntu-trusty/control]: There is a typo `columnAr`.

We might want to highlight, that it's the cpp implementation.
> C++ library to read and write Apache Parquet columnar storage format

**Review by kszucs — APPROVED** [review]
Thanks @kou!

File: dev/tasks/linux-packages/debian.ubuntu-trusty/control (hunk 1 above)
Comment (kou) [reply dev/tasks/linux-packages/debian.ubuntu-trusty/control]: Oh! Thanks for fixing the typo.

C++ information will be useful. We'll add it when we add packages for GLib bindings.

**Pull request closed without merging**